### PR TITLE
session management for remote access client

### PR DESCRIPTION
**Pull request by Aceix**
## Describe your changes
Add auto disabling and enabling feature for RAC sessions

## Provide Issue ticket number if applicable/not in title
NET-584

## Checklist before requesting a review
- [ ] My changes affect only 10 files or less.
- [ ] I have performed a self-review of my code and tested it.
- [ ] If it is a new feature, I have added thorough tests, my code is <= 1450 lines.
- [ ] If it is a bugfix, my code is <= 200 lines.
- [ ] My functions are <= 80 lines.
- [ ] I have had my code reviewed by a peer.
- [ ] My unit tests pass locally.
- [ ] Netmaker is awesome.

### CODE REVIEW

**Review by mattkasun — APPROVED** [review]

**Review by uGiFarukh — APPROVED** [review]

File: controllers/user.go
Comment (abhishek9686) [inline controllers/user.go:1]: You need use the jwt validity from config when creating the jwt token as well for the user

File: controllers/user.go [hunk 1]
```diff
@@ -106,6 +106,33 @@ func authenticateUser(response http.ResponseWriter, request *http.Request) {
...
+			// enable all associeated clients for the user
+			clients, err := logic.GetAllExtClients()
+			if err != nil {
+				slog.Error("error getting clients: ", "error", err)
+				return
+			}
+			for _, client := range clients {
+				if client.OwnerID == username {
+					slog.Info(fmt.Sprintf("enabling ext client %s for user %s due to RAC autodisabling feature", client.ClientID, client.OwnerID))
+					if newClient, err := logic.ToggleExtClientConnectivity(&client, true); err != nil {
```
[the remark this answers is not in the document]
Comment (Aceix) [reply controllers/user.go:121]: resolved

File: controllers/user.go
Comment (Aceix) [reply controllers/user.go:1]: this is node in logic/jwts.go L57

File: pro/remote_access_client.go [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+	clients, err := logic.GetAllExtClients()
+	if err != nil {
+		slog.Error("error getting clients: ", "error", err)
+		return err
+	}
+
+	currentTime := time.Now()
+	validityDuration := servercfg.GetJwtValidityDuration()
+	for _, user := range users {
+		if !currentTime.After(user.LastLoginTime.Add(validityDuration)) {
```
[the remark this answers is not in the document]
Comment (Aceix) [reply pro/remote_access_client.go:43]: resolved

**Review by theguy951357 — APPROVED** [review]
tested and passsed by rohit

**Pull request merged**